parsing/tests/parsing.spec.ts

```diff
@@ -236,5 +254,54 @@ describe('common', () => {
             expect(recipe.building.maxPower).toBe(2000);
             expect(recipe.isAlternate).toBe(false);
         });
+
+
+        it('validate a biomass power generation recipe (solid biomass)', () => {
```

Comment:
```suggestion
        it('should create a biomass power generation recipe (solid biomass)', () => {
```
The `it` function is designed to be more natural language. Grammar nitpick, but that's the idea of `it`